_component_design/assisted-search.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Assisted Search
+parent: Search and Filters
+layout: component
+category: Components
+usage: >
+  Use assisted search when dealing with a complex data set.  Assisted search provides users the ability to see available facets or filters within the search field, allowing for custom construction of search criteria before execution. 
```

Comment:
remove leading indent and extra space after period.

By using assisted search, users can see the available facets or filters within the search field and construct their own search criteria.

Reply:
updated.